src/style.css

```diff
@@ -0,0 +1,41 @@
+@font-face {
+  font-family: Avenir, Times, serif;
+  src: url(./fonts/Avenir.ttc),
+    url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf);
+}
```

Comment:
font-faces should be assigned to some font-weight so that we know when to apply some font, those could be declared the following way
```
@font-face {
  font-family: Avenir;
  src: url(./fonts/Avenir-Book.ttf);
  font-weight: normal;
}

@font-face {
  font-family: Avenir;
  src: url(./fonts/Avenir-Heavy.ttf);
  font-weight: bold;
}
```